src/websocket/client.ts

```diff
@@ -1,3 +1,4 @@
+import URLSearchParams from '@ungap/url-search-params'
```

Comment:
I suggest that instead adding an extra dependency, you change the code in `generateWsUrl` to use `params.append`, instead of `params.set` - the former seems to be implemented in RN - https://github.com/facebook/react-native/blob/master/Libraries/Blob/URL.js#L65.